gdcdictionary/schemas/case.yaml

```diff
@@ -83,7 +83,11 @@ properties:
 
   index_event:
     description: The event used as the anchor or start date for all temporal data elements. This acts as day 0 on a timeline of events relating to the patient.
-    type: string
+    enum:
+      - COVID 19-Test
```

Comment:
Should this be "COVID-19 Test"

Reply:
Updated here and on the voting page. Should be good now - thanks for catching this!!